Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -124,6 +125,20 @@ namespace ROS2
             return;
         }
 
+        AZ::Quaternion rotation(
```

Comment:
```suggestion
        const AZ::Quaternion rotation(
```
Would it be a good idea to reuse this AZ::Quaternion object in the line 212? One duplicated piece of code less.

Reply:
Thank you for the review, I will add it to #762